cosmoz-tabbed-template-behavior.html

```diff
@@ -0,0 +1,109 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../cosmoz-templatize/cosmoz-templatize.html">
+
+<script>
+/**
+ `Cosmoz.TabbedTemplateBehavior`
+ */
+(function () {
+	'use strict';
+
+	window.Cosmoz = window.Cosmoz || {};
+
+	/**
+	* @polymerBehavior Cosmoz.TabbedTemplateBehavior
+	*/
+	Cosmoz.TabbedTemplateBehaviorImpl = {
+		properties: {
+
+			/**
+			 * If it is set to true the template will be stamped
+			 */
+			prerender: {
```

Comment:
What's the difference between a prerendered template and a direct child?
Seems like the use cases would be the same?